### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by imaspacecat**
hi

### CODE REVIEW

File: _posts/2021-09-24-first-post.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: First Post
+date: 2021-09-24 23:10
+categories: jekyll update
+layout: default
+description: The first post on this site...
+---
+
+This is the first post!
```
Comment (airsquared) [inline _posts/2021-09-24-first-post.md:9]: Add another exclamation mark

File: about.markdown [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: About Us
+permalink: /about/
+layout: default
+description: Find out about us!
```
Comment (airsquared) [inline about.markdown:5]: Change to "Find out more about us!"

File: about.markdown [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Last year (2020-2021), our sister team 13190 did not participate in FTC because of a lack of participants. But, this year we have an enormous new influx of ecstatic members who with our help have revived 13190 and are participating this year in FTC Freight Frenzy (whoo hoo!).
+
+Our team organization has remained about the same over the years. We have three departments: hardware, software, and business. Our departments all work very closely with each other in order to construct a competitive robot. For example, the hardware team often helps the software team build a test chassis that can be used for learning and checking code. This allows our mechanical team to work on the main robot while the software team tests code with the extra chassis leading to much progression. Our business team (with the help of software) manages this website, keeps track of expenses, and advises the rest of the team with proper ways to use our funds. They also work on sending emails to local companies for funding and sponsorships.
```
Comment (airsquared) [inline about.markdown:12]: Nice description.

**Review by airsquared — CHANGES_REQUESTED** [review]
Overall looks good, requested some changes before merging.

File: about.markdown (hunk 2 above)
Comment (imaspacecat) [reply about.markdown:5]: Ok

File: about.markdown (hunk 3 above)
Comment (imaspacecat) [reply about.markdown:12]: Thank you!!

**Review by airsquared — APPROVED** [review]

**Pull request merged**